content/versions.md

```diff
@@ -125,13 +125,7 @@ announcement](https://blog.chef.io/2019/04/02/chef-software-announces-the-enterp
 </tr>
 <tr class="odd">
 <td>Chef Workstation</td>
-<td>0.4+</td>
-<td>GA</td>
-<td>n/a</td>
-</tr>
-<tr class="even">
-<td>ChefDK</td>
-<td>4.x</td>
+<td>20.5+ (May 2020)</td>
```

Comment:
This is 20.6+ now

Reply:
Changed accordingly, but I assume that's what the + sign was to account for.